platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1250,11 +1251,17 @@ private Response continueLogout(
         entityId = logoutState.getOriginalIssuer();
         String status =
             logoutState.isPartialLogout() ? StatusCode.PARTIAL_LOGOUT : StatusCode.SUCCESS;
-        logoutObject =
+        LogoutResponse logoutResponse =
             logoutMessage.buildLogoutResponse(
                 SystemBaseUrl.constructUrl(IDP_LOGOUT, true),
                 status,
                 logoutState.getOriginalRequestId());
+        logoutResponse.setDestination(
```

Comment:
✏️ You could still set this to be `logoutObject` to avoid the extra variable and just add a cast here.

Reply:
`setDestination` returns void

Reply:
I meant...
```java
logoutMessage = logoutMessage.buildLogoutResponse(
    SystemBaseUrl.constructUrl(IDP_LOGOUT, true),
    status,
    logoutState.getOriginalRequestId());

((LogoutResponse)logoutMessage).setDestination(
getServiceProvidersMap()
                 .get(entityId)
                 .getLogoutService(SamlProtocol.Binding.HTTP_POST)
                 .getUrl()));
```